README.md

```diff
@@ -242,92 +238,61 @@ helm init
 
 ```bash
 helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 ```
 
 ### 8. Add Local DNS
 
-Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always `en0` as illustrated.  You can use the command `ifconfig -a` to find an active interface.
+We will be forming a local dns with the use of nip.io. All you have to do it get your ip using the following command.
 
 ```bash
-sudo sh -c 'echo "`ipconfig getifaddr en0`       localhost-k8s" >> /etc/hosts'; cat /etc/hosts
+ipconfig getifaddr en0
 ```
 
-*Note:* If your IP address changes you will need to update the `/etc/hosts` entry for localhost-k8s.
+*Note:* Save this ip for later use.
 
 ### 9. Docker Registry Pull Secrets
 
 See the Anaxes Shipyard documentation on [secrets](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md).
 
 *Note*: You can reuse the secrets.yaml file from charts/incubator directory.  
 
-### 10. Apply Alfresco Process Services license
+### 10. Download and modify the minimal-values file
+
+Pull the minimal values file from this repo:
```

Comment:
The distinction between minimal values and "full" values file is nowhere explained. If customers are meant to only use the minimal values file, what for do we need the "full" one?

Reply:
Good point. I'll add the explanation for it.